app/src/main/kotlin/com/wire/android/ui/home/conversations/model/UIMessage.kt

```diff
@@ -109,18 +109,19 @@ sealed class ExpirationStatus {
 sealed class MessageStatus(
     open val errorText: UIText? = null, // error description text shown below the content of the message
     open val badgeText: UIText? = null, // text shown between the user name and the content in the outlined box with a text inside
-    open val isPending: Boolean = false // if true then such message is still being sent
+    open val status: Message.Status = Message.Status.SENT
```

Comment:
I'm not a fan of this change, because now we can possibly have `Edited` with status `FAILED` inside, which should be `EditSendFailure`.
I think here it's better to keep it simple, by creating an enum class for state I meant a new different one just for the `MessageStatusIndicator` so that's independent from the `message` object and its parameters completely.

Reply:
refactored